### DISCUSSION TITLE
Day/Night Cycle Plugin!

### DISCUSSION THREAD

**Discussion by anaMadeline** [runelite/runelite#15388] [Ideas]
I've been looking into making a Day/Night cycle plugin, what are your thoughts about it?

My first idea is along the lines of a "Night Shader", today I cloned the repo and got into the shaders, folder and even had a little bit of fun fiddling with the colorBlindMode, I added a new option for the "Night mode" and made everything turn green, but of course that's not the right route. So now I'm looking even deeper into the GPU plugin to see what I can tweak, but I'm starting to think it will be a lot more interesting that that.

What do you all think about it? I'm just now getting acquainted with the source code, and with java for that matter, this would be my first thing here. So I'm looking for some heads ups and recommendations of where and what to look for, what to fiddle with to get my hands dirty with RuneLite's GPU plugin. I'll also take a look at 117scape's project soon for inspiration, his creation is an absolute wonder, and in a way what I have in my head may be like that, "a GPU plugin that adds more interesting things", nothing more, at first I just want to look like it's night in the game.

Tomorrow I'll see what else I can do, my first idea is to just clone the GPU plugin and start changing things maybe adding some sliders and see what sticks, I still need to understand how RuneLite works after all.

**Comment by MrGuanaco** [comment]
Hello, have you made any progress?
